config/validateRelease.py

```diff
@@ -37,6 +37,7 @@
 
 import argparse
 import os
+import sets
```

Comment:
From the Python docs :

> Deprecated since version 2.6: The built-in set/frozenset types replace this module.

Reply:
Doh! That'll teach me to inadvertently read python 2.3 docs when looking up set details. Thanks. Fixed in https://github.com/GafferHQ/gaffer/commit/b04ce8615ca9b51406534da83b0b3fc32ac3acd7.